<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Terminal</class>
 <widget class="QMainWindow" name="Terminal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Terminal - by Chr1$</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="0" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QPushButton" name="connect_button">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rescan_button">
        <property name="text">
         <string>ReScan</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="help_button">
        <property name="text">
         <string>Help</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="about_button">
        <property name="text">
         <string>About</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="quit_button">
        <property name="text">
         <string>Quit</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <widget class="QGroupBox" name="groupBox_5">
      <property name="title">
       <string>COM Port</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QComboBox" name="com_port_combo"/>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Baud Rate</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="0">
        <widget class="QRadioButton" name="baud_9600_radio">
         <property name="text">
          <string>9600</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <widget class="QRadioButton" name="baud_4800_radio">
         <property name="text">
          <string>4800</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QRadioButton" name="baud_600_radio">
         <property name="text">
          <string>600</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QRadioButton" name="baud_14400_radio">
         <property name="text">
          <string>14400</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="baud_1200_radio">
         <property name="text">
          <string>1200</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QRadioButton" name="baud_19200_radio">
         <property name="text">
          <string>19200</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="baud_2400_radio">
         <property name="text">
          <string>2400</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QRadioButton" name="baud_38400_radio">
         <property name="text">
          <string>38400</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QRadioButton" name="baud_56000_radio">
         <property name="text">
          <string>56000</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QRadioButton" name="baud_57600_radio">
         <property name="text">
          <string>57600</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QRadioButton" name="baud_115200_radio">
         <property name="text">
          <string>115200</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QRadioButton" name="baud_128000_radio">
         <property name="text">
          <string>128000</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QRadioButton" name="baud_256000_radio">
         <property name="text">
          <string>256000</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Data Bits</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QRadioButton" name="data_bits_5_radio">
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="data_bits_6_radio">
         <property name="text">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="data_bits_7_radio">
         <property name="text">
          <string>7</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QRadioButton" name="data_bits_8_radio">
         <property name="text">
          <string>8</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="5" colspan="2">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Stop Bits</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="4" column="0">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="stop_bits_1_radio">
         <property name="text">
          <string>1</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QRadioButton" name="stop_bits_2_radio">
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="stop_bits_1_1_2_radio">
         <property name="text">
          <string>1.5</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="7" colspan="2">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Parity</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QRadioButton" name="parity_none_radio">
         <property name="text">
          <string>none</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="parity_odd_radio">
         <property name="text">
          <string>odd</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="parity_even_radio">
         <property name="text">
          <string>even</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer_11">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <widget class="QRadioButton" name="parity_mark_radio">
         <property name="text">
          <string>mark</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QRadioButton" name="parity_space_radio">
         <property name="text">
          <string>space</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="9">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Receive</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="9">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="clear_button">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="autoscroll_check">
        <property name="text">
         <string>AutoScroll</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QRadioButton" name="hex_rx_radio">
             <property name="text">
              <string>Hex</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="ascii_rx_radio">
             <property name="text">
              <string>ASCII</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="3" colspan="7">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0" colspan="10">
     <widget class="QTextEdit" name="receive_text">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0" rowspan="2">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Transmit</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" rowspan="2" colspan="9">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="6" rowspan="2" colspan="2">
     <widget class="QGroupBox" name="groupBox_7">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QRadioButton" name="hex_tx_radio">
           <property name="text">
            <string>Hex</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="ascii_tx_radio">
           <property name="text">
            <string>ASCII</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="6" column="0" colspan="6">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="set_macro_button">
        <property name="text">
         <string>Set Macros</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="m2_button">
        <property name="text">
         <string>M2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="m3_button">
        <property name="text">
         <string>M3</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="m4_button">
        <property name="text">
         <string>M4</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="m5_button">
        <property name="text">
         <string>M5</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QPushButton" name="m6_button">
        <property name="text">
         <string>M6</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="m7_button">
        <property name="text">
         <string>M7</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="m8_button">
        <property name="text">
         <string>M8</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="m9_button">
        <property name="text">
         <string>M9</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="m10_button">
        <property name="text">
         <string>M10</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="m11_button">
        <property name="text">
         <string>M11</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QPushButton" name="m12_button">
        <property name="text">
         <string>M12</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="m1_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>M1</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="6" column="8" colspan="2">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>196</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="0" colspan="10">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="transmit_field">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="send_button">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="8" column="0" colspan="10">
     <widget class="QTextEdit" name="transmit_text">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>765</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="toolTip">
    <string/>
   </property>
   <property name="statusTip">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
